Documentation/glosary

TomasArguelles opened the pull request:
No se me ocurren términos para añadir, que puedan colar, decidmelos si se os ocurren

TomasArguelles committed:
Añadidos terminos al glosario 1

TomasArguelles committed:
Update 12_glossary.adoc

TomasArguelles committed:
Update 12_glossary.adoc

acg96 commented:
Voy a empezar detallando los errores que he visto, en la descripción de que el POD dentro de poco podrá estar en tu propio ordenador y demás, eso ya es posible siempre que montes tu el servidor. De forma simple no está pensado que se vaya a hacer. 
Luego en el tema de SOLID también estamos siguiendo los principios SOLID tradicionales en gran medida a parte de los del propio proyecto SOLID.

Luego de los errores, yo pondría muchas mas cosas, es mejor que mires el resto de documentación y todos los tecnicismos que encuentres los detalles, por ejemplo faltan cosas importantes como que es Linked Data, RDF, que significa descentralización, arquitectura en capas, MVC... entro otros muchos

TomasArguelles committed:
Update 12_glossary.adoc

añadidas sugerencias al glosario, faltan más

enolgargon commented:
Yo creo que no entiendes lo que ye un glosario. Hay que definir los términos en función al uso que se da en el documento. Que importa que SOLID puedan ser los principios. Y la definición de Linked Data paez una campaña comercial. 
Yo no inluiría ningún link.
La definición de centralización no me gusta absolutamente nada.
POD se podría separar en dos: POD y solid server. y explicalo mejor
RDF falta mucha información, solo pones que ye un estandar, pon que se basa en construir oraciones con linked data o algo asi.
MVC muy bien definido, pero si soy el cliente me preguntaría que ye un modelo y un controlador. La vista a lo mejor lo saco pensando, pero lo otro ni idea.

Y falten definiciones por un tubo: Angular, Node, Typescript, API Rest, petición web, mock up, etc...

TomasArguelles committed:
Update 12_glossary.adoc

Cambiada definición de POD, SOLID, Linked Data, descentralizacion; añadidas aclaraciones en MVC; añadida definición de Semantic Queries, SOLID server (No encuentro por ningún lado nada sobre las puñeteras especificaciones solo que existen) API, REST, API Rest, Web Petition, Mock up; eliminados links

TomasArguelles commented:
Algo mejor?

TomasArguelles committed:
Update 12_glossary.adoc

Added ARQUISOFT, Version control, DeChat, Uri, Cross-cutting concepts, Principle of single responsibility, open-closed principle and Dependency inversion principle and typos

enolgargon commented:
Vale, me gusta mas. Algunos cambios:
- Rest y Api Rest juntalo
- Añade les deficioniones de Card (el lugar del pod en el que se almacena la inforación sobre el usuario) y Inbox, el lugar del pod en el que el resto del mundo pue escribir, per no leer ni modificar y que se usa pa les notificaciones.
- Creo que falta en algún sitiu mas de la documentación. Pa los test usamos Karma y Cucumber. Y pa poder ejecutar Cucumber en Angular tenemos un driver de Selenium y Protractor. Define Selenium, Protractor, Karma, Cucumber, testing, y mira a ver si falta por poner información de testing en la documentación

TomasArguelles committed:
Update 12_glossary.adoc

Fusionadas Api Rest y REST y añadidas Card, Inbox, Karma, Cucumber, Selenium, Protractor and Testing

enolgargon commented:
Mejor, yo lo veo bien. A ver que diz Andrés
